client/components/product-list.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ProductListItem from './product-list-item.jsx';
+
+export default class ProductList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      products: []
+    };
+  }
+
+  getProducts() {
+    fetch('http://localhost:3000/api/products', {
```

Comment:
You should not hard-code the origin in your url. you can just put `/api/products`

Reply:
![image](https://user-images.githubusercontent.com/61432718/88346975-839bb280-ccfe-11ea-9f5a-f4e0a648ffa8.png)